src/test/test-execute.c

```diff
@@ -1115,6 +1180,9 @@ static void test_exec_networknamespacepath(Manager *m) {
         if (!have_netns)
                 return (void)log_notice("Skipping %s, network namespace not available", __func__);
 
+        if (MANAGER_IS_USER(m) && !have_unprivileged_userns)
+                return (void)log_notice("Skipping %s, cannot create unprivileged user namespaces", __func__);
```

Comment:
these log message should also be updated, as per the comment son "unprivileged" above.